WordPress/gutenberg
# [RNMobile] Add setMinHeightToMaxHeight prop to bottom sheet

## Description
This adds the ability to set the `minHeight` equal to the `maxHeight` of the `BottomSheet` component. This is useful when trying to render dynamic content within the bottom sheet. The current need is for [Inserter Block Search](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2570)  project. 

## How has this been tested?
Note: The Inserter Block Search is currently behind the `__DEV__` flag. 
1. Start the Mobile Gutenberg demo app
2. Open the mobile editor
3. Open the block inserter menu
4. Press on "Search Blocks"
5. Enter a search query that returns few results e.g. `img`
6. The bottom sheet height should not change.

## Screenshots 

Before:

Android | iPhone
:---: | :---:
![](https://user-images.githubusercontent.com/744755/108920686-f305b780-75e9-11eb-8857-e07a7404921a.gif) | ![](https://user-images.githubusercontent.com/744755/108920880-4e37aa00-75ea-11eb-9abb-fd98b081dae3.gif)


After:

Android | iPhone
:---: | :---:
![](https://user-images.githubusercontent.com/744755/108920934-6a3b4b80-75ea-11eb-8e16-e80807eb47d9.gif) | ![](https://user-images.githubusercontent.com/744755/108921040-99ea5380-75ea-11eb-89ac-5a089b02b98b.gif)


## Types of changes
New feature (non-breaking change which adds functionality) 
- Adds a `setMinHeightToMaxHeight` prop  to the `BottomSheet` component. When `true` the bottom sheet will have a static height equal to the max height. example:

```jsx
<BottomSheet setMinHeightToMaxHeight={ true } > 
```


## Checklist:
